### ISSUE TITLE
TypeError: Cannot read property 'r' of undefined at interpolateRGBAinCieLAB

### ISSUE THREAD

**Issue by rochoa** [CartoDB/carto-vl#1270]
```
colorspaces.js:12 Uncaught (in promise) TypeError: Cannot read property 'r' of undefined
    at interpolateRGBAinCieLAB (colorspaces.js:12)
    at mix (utils.js:231)
    at RampGeneric._calcEval (RampGeneric.js:66)
    at RampGeneric.eval (RampGeneric.js:48)
    at Dataframe._computePointRadius (Dataframe.js:347)
    at Dataframe._getPointsAtPosition (Dataframe.js:323)
    at Dataframe.getFeaturesAtPosition (Dataframe.js:92)
    at rawFeatures.concat.getActiveDataframes.map.df (RenderLayer.js:88)
    at Array.map (<anonymous>)
    at RenderLayer._getRawFeaturesAtPosition (RenderLayer.js:87)
```

![screen shot 2019-02-21 at 20 22 05](https://user-images.githubusercontent.com/45346/53196094-6baa9580-3617-11e9-8b7f-37c2990a9d29.png)

Snippet to reproduce it:

```js
const map = new mapboxgl.Map({
    container: 'map',
    style: carto.basemaps.voyager,
    center: [-3.723, 40.407],
    zoom: 15
});

carto.setDefaultAuth({
    username: 'cartovl',
    apiKey: 'default_public'
});
const source = new carto.source.Dataset('madrid_listings');
const viz = new carto.Viz(`
    width: ramp(linear($price, 0, 40), [8, 32])
`);

const layer = new carto.Layer('layer', source, viz);
layer.addTo(map);

const interactivity = new carto.Interactivity([layer]);
```

**Comment by jorgesancha** [comment]
One for the RT, I think. cc  @VictorVelarde  @CartoDB/rt-managers

**Comment by VictorVelarde** [comment]
We have applied a fix, just in time to be included in the next v1.2 release.

**Comment by jorgesancha** [comment]
I believe we can close this, right @VictorVelarde ?
